client/extensions/woocommerce/state/sites/customers/selectors.js

```diff
@@ -1,86 +1,58 @@
-/**
- * External dependencies
- */
-
-import { filter, get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+function inFlight( state, siteId, searchTerm ) {
+	return state?.extensions?.woocommerce?.sites?.[ siteId ]?.customers.isSearching[ searchTerm ];
```

Comment:
If the `woocommerce` is there, the `sites` tree will always be there, so maybe the optional chaining for `?.sites` can be removed? This has multiple occurrences in this PR.

Reply:
That would mean that we could remove the `?.[ siteId ]` chaining, not the `?.sites` one, right? The conditional in `?.` is applied to what precedes it, not what succeeds it.

In any case, I had tried assuming that `sites` was always there, but the tests failed, which is why I dropped it. On second inspection, though, it appears to be because the tests were incorrect. Fixed!

Reply:
Hah, yeah, typo is on me here 👍 Thanks for fixing